client/lib/i18n-utils/switch-locale.js

```diff
@@ -295,10 +295,15 @@ function addRequireChunkTranslationsHandler( localeSlug = i18n.getLocaleSlug(),
 			} )
 			.catch( ( cause ) => {
 				const error = new Error(
-					`Encountered an error loading translation chunk ${ chunkId } for "${ localeSlug }" in require chunk translations handler.`,
+					`Encountered an error loading translation chunk in require chunk translations handler.`,
 					{ cause }
 				);
-				captureException( error );
+				captureException( error, {
+					tags: {
+						chunk_id: chunkId,
+						locale_slug: localeSlug,
+					},
+				} );
```

Comment:
Nit: This is an opportunity for extracting to a separate function since it's duplicated in `switchLocale()` below.